Sources/Inject/Inject.swift

```diff
@@ -7,22 +7,19 @@ import SwiftUI
 public protocol InjectListener {
     associatedtype InjectInstanceType = Self
 
-    func enableInjection()
     func onInjection(callback: @escaping (InjectInstanceType) -> Void) -> Void
 }
 
 /// Public namespace for using Inject API
 public enum Inject {
     public static let observer = injectionObserver
-    public static let load: Void = loadInjectionImplementation
```

Comment:
Is it necessary to deprecate this released api?

Reply:
Not necessary, but I wonder if we should keep it if it happens automatically?

Reply:
I restored this method, but added "deprecated" warning instead for now :)